calculate-returns.py

```diff
@@ -0,0 +1,61 @@
+# Python script to calculate the pension fund for mutual funds
+import pandas as pd
+
+# Import the dataframe from excel file
+df_pension = pd.read_excel('./data/16-Apr-2021.xls', header=None)
```

Comment:
Could you take the xls file as a command line arg? And I'm not sure we want it committed in the repo.

Reply:
Hear me out... how about:
1. We use `datetime` library to fetch last friday's report. For last week's report it was of format: http://www.npstrust.org.in/sites/default/files/07-May-2021_0.xls (Darn, buggers have added this extra _0 to the URL that deviates from usual naming convention)
2. Convert this report into CSV/JSON, whatever you feel comfortable with.
3. Commit this CSV file, or rather overwrite the CSV file in data folder.
4. Plot the graphs using the updated data. Plot would get updated based on latest data.

This is my reasoning behind this approach:
1. We will have an archive of these weekly report, in forms of git commits.
2. Converting XLS format to more open, accessible, text based formatting is GOOD.

Thoughts?

Reply:
I think my comment doesn't help. It is complicated the task and scope of the PR. Lets keep it simple, either way is fine, command line option or committing the file.